app/src/main/java/com/compscieddy/meetinthemiddle/Util.java

```diff
@@ -40,17 +40,9 @@ public static Bitmap getCroppedBitmap(Context context, Bitmap bitmap) {
     return output;
   }
 
-  public static void rotateFabForward(View view) {
+  public static void rotateView(View view, float angle){
```

Comment:
@eddieberklee I thought this function could be used for rotating any views in the future, not just the FAB, which is why I renamed it to rotateView.
Also, I'm using this function with different angles in the code, which is why the angle parameter is needed.
What do you mean by this should be a boolean? This is the function to carry out the rotation, why do I need a boolean?